src/winrpath.cxx

```diff
@@ -286,6 +286,11 @@ void CoffReaderWriter::write(char * in, int size)
     this->pe_stream.write(in, size);
 }
 
+void CoffReaderWriter::flush()
+{
+    this->pe_stream.flush();
+}
```

Comment:
I don't see anything calling this function, which I think is new

Reply:
I found it quite useful for debugging, and thought that enough reason to leave it in. Happy to remove it if you disagree.